yarn-project/sequencer-client/src/test/l2-block-publisher.test.ts

```diff
@@ -65,6 +210,9 @@ async function deployRollup() {
   const ethRpc = new EthereumRpc(provider);
 
   // Deploy Rollup and unverifiedDataEmitter contracts
+  const decodeHelper = new DecoderHelper(ethRpc, undefined, { from: deployer, gas: 1e6 });
+  await decodeHelper.deploy().send().getReceipt();
```

Comment:
would rename `decodeHelper` to `decoderHelper` for consistency + please update the comment above.